### PR TITLE
Update BBEs for 0.991.0 - PR 2

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

### CODE REVIEW

File: examples/any-type/any_type.description [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // The `any` type can represent a variable of any type, except the `error` type.
 // When you do not have prior knowledge of the type of a variable, you can assign it to a variable of type `any`.
 // Values of these variables can come from dynamic content such as the request and response message references and
-// Ballerina reference variable types. The `any` type allows you to skip compile-time type checks.
+// Ballerina reference variable types.
```
Comment (pubudu91) [inline examples/any-type/any_type.description]:
```suggestion
// reference variable types.
```

File: examples/optional-type/optional_type.bal [hunk 2]
```diff
@@ -1,16 +1,15 @@
 import ballerina/io;
 
-//This function optionally returns a `string` value.
-function getValue(string key) returns string|() {
+//This function optionally returns a `string` value. `?` here is equivalent to `|()`.
```
Comment (pubudu91) [inline examples/optional-type/optional_type.bal]: Shall we rephrase the last bit to something like this: "`string?` is equivalent to `string|()`."?

Comment (pubudu91) [reply examples/optional-type/optional_type.bal]:
```suggestion
//This function optionally returns a `string` value. `string?` is equivalent to `string|()`.
```

File: examples/record-destructure-binding-pattern/record_destructure_binding_pattern.description [hunk 3]
```diff
@@ -1,3 +1,3 @@
-// Record de-structure binding patterns are used to de-structure and assign the values of each field in the destructed record to variable references.
-// The record destructuring binding patterns will be used to refer existing variables as a record and de-structure the
-// the given value on the right hand side and assign the values to each individual variables of the record at runtime.
+// Record destructure binding patterns are used to destructure and assign the values of each field in the destructed record to variable references.
+// The record destructuring binding patterns will be used to refer existing variables as a record and destructure the
```
Comment (pubudu91) [inline examples/record-destructure-binding-pattern/record_destructure_binding_pattern.description]:
```suggestion
// The record destructuring binding patterns will be used to refer to existing variables as a record and destructure the
```

File: examples/tuple-destructure-binding-pattern/tuple_destructure_binding_pattern.description [hunk 4]
```diff
@@ -1,3 +1,3 @@
-// Tuple binding patterns are used to de-structure and assign values to each individual variables of the tuple.
-// The tuple destructuring binding patters will be used to refer existing variables as a tuple and de-structure the
+// Tuple binding patterns are used to destructure and assign values to each individual variables of the tuple.
+// The tuple destructuring binding patters will be used to refer existing variables as a tuple and destructure the
```
Comment (pubudu91) [inline examples/tuple-destructure-binding-pattern/tuple_destructure_binding_pattern.description]:
```suggestion
// The tuple destructuring binding patterns will be used to refer to existing variables as a tuple and destructure the
```

File: examples/record-destructure-binding-pattern/record_destructure_binding_pattern.description (hunk 3 above)
Comment (MaryamZi) [reply examples/record-destructure-binding-pattern/record_destructure_binding_pattern.description]: Again not sure we need "to" here.

File: examples/tuple-destructure-binding-pattern/tuple_destructure_binding_pattern.description (hunk 4 above)
Comment (MaryamZi) [reply examples/tuple-destructure-binding-pattern/tuple_destructure_binding_pattern.description]: Will change to 
```suggestion
// The tuple destructuring binding patterns will be used to refer existing variables as a tuple and destructure the
```
for the time being.

**Comment by praneesha** [comment]
Issue #14969 created for the doc review of this.

**Pull request merged**